Framework/WorkflowAlgorithms/src/AlignAndFocusPowder.cpp

```diff
@@ -414,8 +396,8 @@ void AlignAndFocusPowder::exec() {
       double tempmax;
       m_outputW->getXMinMax(tempmin, tempmax);
 
-      g_log.information() << "running CropWorkspace(Xmin=" << xmin
-                          << ", Xmax=" << xmax << ")\n";
+      g_log.information() << "running CropWorkspace(TOFmin=" << xmin
+                          << ", TOFmax=" << xmax << ")\n";
```

Comment:
If you changing the log info, would renaming `xmin` and `xmax` make them more descriptive?

The coding standards also say these member functions should have an `m_` prefix.